q2_gneiss/plot/_plot.py

```diff
@@ -130,20 +215,27 @@ def f(x):
 
 plugin.visualizers.register_function(
     function=balance_taxonomy,
-    inputs={'balances': FeatureTable[Balance], 'tree': Hierarchy,
+    inputs={'table': FeatureTable[Composition], 'tree': Hierarchy,
             'taxonomy': FeatureData[Taxonomy]},
     parameters={'balance_name': Str,
                 'taxa_level': Int,
-                'metadata': MetadataCategory},
+                'metadata': MetadataCategory,
+                'n_features': Int,
+                'threshold': Float},
     input_descriptions={
-        'balances': 'The table of balances resulting from the ilr transform.',
+        'table': 'A table of compositions.',
         'tree': 'The tree used to calculate the balances.',
-        'taxonomy': 'Taxonomy information for the OTUs.',
+        'taxonomy': 'Taxonomy information for the OTUs.'
     },
     parameter_descriptions={
         'balance_name': 'Name of the balance to summarize.',
         'taxa_level': 'Level of taxonomy to summarize.',
-        'metadata': 'Metadata for plotting the balance (optional).'},
+        'metadata': 'Metadata for plotting the balance (optional).',
+        'n_features': 'The number of features to plot in the proportion plot.',
+        'threshold': ('A threshold to designate discrete categories '
+                      'for continuous metadata.  If not specified, '
```

Comment:
This was pretty confusing until I realized it was splitting your metadata into two groups. Maybe mentioning that would be good, and also whether the behavior is inclusive or exclusive with respect to the threshold and the first group.

Reply:
Updating the `threshold` parameter as 
```
        'threshold': ('A threshold to designate discrete categories '
                      'for numerical metadata.  This will split the '
                      'numerical category into two categories, values below '
                      'the threshold, and values above the threshold.'
                      'If not specified, this threshold will '
                      'default to the mean.')
```